### PR TITLE
Modify feature-flag analytics shape, include browser language

### PR DESCRIPTION
**Pull request by Jakedavies**
Note there is a change here with the way the feature flags work:

We now send the entire key->value array blob to the analytics systems.

### CODE REVIEW

**Comment by Jakedavies** [comment]
Need to update the tests

File: src/featureFlags.ts [hunk 1]
```diff
@@ -80,15 +80,7 @@ export function featureFlagIsEnabled (name: string): boolean {
...
-    const featureFlagArray = [];
-    for (const featureFlag in cachedFeatureFlags) {
-        if (cachedFeatureFlags[featureFlag]) {
-            featureFlagArray.push(featureFlag);
-        }
-    }
-
     analytics.setUserProperties({
-        featureFlags: featureFlagArray,
+        featureFlags: cachedFeatureFlags,
```
Comment (armanreyes3) [inline src/featureFlags.ts:84]: does this still work for amplitude?

**Review by aduros — APPROVED** [review]
Please make sure PMs know about this change for Amplitude before merging

File: src/featureFlags.ts (hunk 1 above)
Comment (Jakedavies) [reply src/featureFlags.ts:84]: Good catch, it didn't. Added a backwards compatibility function for this.

**Comment by Jakedavies** [comment]
> Please make sure PMs know about this change for Amplitude before merging

This is no longer a change for amplitude, the amplitude layer now has a backwards compat pass to keep the shape the same since amplitude doesn't support the key value format at all.

**Review by armanreyes3 — APPROVED** [review]

**Review by aduros — APPROVED** [review]
Nice!

**Comment by aduros** [comment]
@Jakedavies Is this still needed?

**Comment by Jakedavies** [comment]
> @Jakedavies Is this still needed?

Yes, I'll clean up the conflict and get it merged. I forgot about it

**Pull request merged**